test/isArray_test.js

```diff
@@ -0,0 +1,65 @@
+goog.module('datalayerhelper.helper.testing.isArray_');
+goog.setTestOnly();
+
+const {isArray_} = goog.require('helper');
+
+describe('The `isArray_` function of helper', () => {
+  /**
+   * Ensure that plain.IsPlainObject recognizes value
+   * to be a array.
+   * @param {*} value The value to check
+   */
+  function assertIsArray(value) {
+    expect(isArray_(value)).toBe(true);
+  }
+
+  /**
+   * Ensure that plain.IsPlainObject doesn't recognize
+   * the value to be an array.
+   * @param {*} value The value to check.
+   */
+  function assertIsNotArray(value) {
+    expect(isArray_(value)).toBe(false);
+  }
+
+  it(`identifies things that aren't arrays as not arrays`, () => {
```

Comment:
nit: tick to quote, please.

Reply:
aren't has a quote in it, this needs ticks

Reply:
Ah. Maybe we should switch all of these test descriptions to double quotes?

... jk.